app.js

```diff
@@ -49,18 +48,45 @@ function Runner(
                 return skuPack.start(configuration.get('skuPackRoot', './skupack.d'));
             })
             .then(function() {
-                return app.listen();
+                // TODO: The following endpoint definition is for DEBUG using
+                // It will fianlly been moved to monorail.json config file
+                // var endpoints = configuration.get('httpEndpoints', [{port:8080}]);
```

Comment:
Yes, I have removed the local definition of `var endpoints`, it will get from `monorail.json`